import-url/update: add --no-download flag (#3773)
iterative/dvc.org: content/docs/command-reference/update.md

## Synopsis

```usage

> usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R] [--no-download] [--to-remote]
>                   [-r <name>] [-j <number>] targets [targets ...]

Reviewer: Here too...
```suggestion
usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R]
                  [--no-download] [--to-remote] [-r <name>]
                  [-j <number>]
                  targets [targets ...]
```

## Synopsis

```usage
usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R]
                  [--no-download] [--to-remote] [-r <name>]
                  [-j <number>]

>                   targets [targets ...]

Author: Not sure what you mean here

  directory and its subdirectories for import `.dvc` files to inspect. If there
  are no directories among the targets, this option has no effect.

- `--no-download` - Update data checksums in the `.dvc` file (`md5`, `etag`, or
  `checksum` fields) without actually downloading the latest data. See

>   `dvc import-url --no-download`/`dvc import --no-download` for more context.

Reviewer: 💅🏼 
```suggestion
  `dvc import-url --no-download` or `dvc import --no-download` for more context.
```

  directory and its subdirectories for import `.dvc` files to inspect. If there
  are no directories among the targets, this option has no effect.

> - `--no-download` - Update hash values in the `.dvc` file (`md5`, `etag`, or

Reviewer 2: Very minor, but is it hash values or checksums, and can we be consistent?

  `checksum` fields) without actually downloading the latest data. See
  `dvc import-url --no-download`/`dvc import --no-download` for more context.
  Cannot be combined with `--to-remote`.

- `--to-remote` - update a `.dvc` file created with `dvc import-url` without
  downloading the latest data.

>   [Transfer it](/doc/command-reference/import-url#example-transfer-to-remote-storage)

Reviewer 2: Let's better differentiate this from `--no-download`. Maybe something like:

```
- `--to-remote` - update a `.dvc` file created with `dvc import-url` and 
  [transfer](/doc/command-reference/import-url#example-transfer-to-remote-storage)
  the data directly to remote storage (the default one unless one is specified with -r)
  without saving it locally. Use [dvc pull](https://dvc.org/doc/command-reference/pull)
  to get the data locally.
```

  directory and its subdirectories for import `.dvc` files to inspect. If there
  are no directories among the targets, this option has no effect.

> - `--no-download` - Update checksums in `.dvc` without actually downloading the
>   associated data. Cannot be used with `--to-remote`.

Reviewer: ```suggestion
- `--no-download` - Update hash values in the `.dvc` file (`md5`, `etag`, or
  `checksum` fields) without actually downloading the latest data. See
  `dvc import-url --no-download` for more context. Cannot be combined with
  `--to-remote`.
```
